packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -90,22 +96,30 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
                 if (!resource) {
                     throw new Error(`${VscodeCommands.OPEN.id} command requires at least URI argument.`);
                 }
+
                 if (!URI.isUri(resource)) {
                     throw new Error(`Invalid argument for ${VscodeCommands.OPEN.id} command with URI argument. Found ${resource}`);
                 }
 
-                let options: TextDocumentShowOptions | undefined;
-                if (typeof columnOrOptions === 'number') {
-                    options = {
-                        viewColumn: columnOrOptions
-                    };
-                } else if (columnOrOptions) {
-                    options = {
-                        ...columnOrOptions
-                    };
+                const stat = await this.fileSystem.getFileStat(new TheiaURI(resource).toString());
```

Comment:
resource is not necessary file URIs, it can be any URI supported by extension, i.e. gitlens contribute special schemes. It would break all such extensions.

Reply:
Any example of such an URI?